google-cloud-tools-plugin/resources/config.properties

```diff
@@ -19,3 +19,5 @@ feature.appengine.flex=true
 
 # Should be used in any Notifications explicitly produced by the plugin.
 notifications.plugin.groupdisplayid=Google App Engine plugin
+
+cloudsdk.min.version=131.0.0
```

Comment:
It seems that several tools panic over the lack of new lines at the end of text files, or add noise when you're closing them.

http://stackoverflow.com/questions/4700312/empty-new-line-at-the-end-of-the-java-source-files